testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/LoggedInPageHeaderPage.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.testsuite.ui.account2.page;
+
+import org.keycloak.representations.idm.UserRepresentation;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static org.keycloak.testsuite.util.UIAssert.assertElementDisabled;
+import static org.keycloak.testsuite.util.UIAssert.assertInputElementValid;
+import static org.keycloak.testsuite.util.UIUtils.getTextInputValue;
+import static org.keycloak.testsuite.util.UIUtils.setTextInputValue;
+
+import static org.junit.Assert.assertEquals;
+
+public class LoggedInPageHeaderPage extends AbstractLoggedInPage {
```

Comment:
Please don't make a separate Page Class for this. It's not a page, it's part of a page header. Please move its content to [AbstractHeader](https://github.com/keycloak/keycloak/blob/f2715025b344d37411701efec2df07376836979e/testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/fragment/AbstractHeader.java) (if the username will be displayed on Welcome Screen as well) or [LoggedInPageHeader](https://github.com/keycloak/keycloak/blob/f2715025b344d37411701efec2df07376836979e/testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/fragment/LoggedInPageHeader.java).